src/v2/Apps/Artist/Routes/Overview/Components/ArtistRelatedGeneCategories.tsx

```diff
@@ -0,0 +1,96 @@
+import { Pill, Spacer, Text } from "@artsy/palette"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "v2/System"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { RouterLink } from "v2/System/Router/RouterLink"
+import { extractNodes } from "v2/Utils/extractNodes"
+import { ArtistRelatedGeneCategories_artist } from "v2/__generated__/ArtistRelatedGeneCategories_artist.graphql"
+import { ArtistRelatedGeneCategoriesQuery } from "v2/__generated__/ArtistRelatedGeneCategoriesQuery.graphql"
+
+interface ArtistRelatedGeneCategoriesProps {
+  artist: ArtistRelatedGeneCategories_artist
+}
+
+const ArtistRelatedGeneCategories: FC<ArtistRelatedGeneCategoriesProps> = ({
+  artist,
+}) => {
+  const categories = extractNodes(artist.related?.genes)
+
+  if (categories.length === 0) return null
+
+  return (
+    <>
+      <Text variant="lg-display">Related Categories</Text>
+
+      <Spacer mt={4} />
+
+      {categories.map(category => {
```

Comment:
Since the pills have an `mb={1}` that means that there'll be a `1` external margin. We can wrap `categories` with `<Box mb={-1}>`  which will offset that last row of margins while still allowing for the internal margins when they break lines.